### DISCUSSION TITLE
Latest stable release for Miyoo mini plus?

### DISCUSSION THREAD

**Discussion by errchh** [OnionUI/Onion#992] [Q&A] [answered]
Question on stable version for Miyoo mini plus. 

Device -- Miyoo Mini plus
Version -- 202305052130 

Sorry for a beginner question. I'm a bit confused about which version I could use for Miyoo mini plus. According to the release page (https://github.com/OnionUI/Onion/releases), 

-- from Onion 4.0.4 onwards, versions are not compatible with the mini plus. But 
-- 4.0.3 and below is NOT COMPATIBLE with newer firmware versions (20221216**** and 20221224****). 

So which stable release I could use for Miyoo mini plus?

**Comment by Aemiii91** [comment, marked as answer]
None of them. Use 4.2-beta for Plus.

**Comment by wendychagas** [comment]
hello.. 
i really wish to know if you know whats is it the latest stable version for miyoo mini plus version 2023.06.09?

**Reply by Aemiii91** [reply]
None. See above.
